content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -661,6 +661,67 @@ of single-resource API requests, then aggregates the responses if needed.
 By contrast, the Kubernetes API verbs **list** and **watch** allow getting multiple
 resources, and **deletecollection** allows deleting multiple resources.
 
+## Field Validation
+
+  {{< feature-state for_k8s_version="v1.24" state="beta" >}}
+
+In certain situations, the API server will drop fields it does not recognize
+from an object it receives.
+
+The two situations this will occur are when:
+
+1. The field is unrecognized, meaning not in the resource's OpenAPI schema. (One
+   exception to this is for CRD's that explicitly choose not to prune unknown
+   fields via `x-kubernetes-preserve-unknown-fields`)
+2. The field is duplicated in the object.
+
+When you use HTTP verbs that can submit data (`POST`, `PUT`, and `PATCH`) field
+validation gives you the option to choose how you would like to be notified of
+these fields that are being dropped by the API server. Possible levels of
+validation are `Ignore`, `Warn`, and `Strict`.
+
+{{< note >}}
+Any fields that are erroneous for other reasons (i.e. a
+field that is supposed to be an int, but is submitted as a string), will always
+result in the request failing with a bad request error.
+{{< /note >}}
+
+### Setting the field validation level
+
+Field validation is set by the `fieldValidation` query parameter. This parameter
+is a string, working as an enum and the only accepted values are:
+
+`Ignore`
+: The API server succeds in handling the request as it would without the erroneous fields
+being set, dropping all unknown and duplicate fields and giving no indication it
+has done so.
+
+`Warn`
+: (Default) The API server succeeds in handling the request, but will add a
+warning to the warning header for each unknown/duplicate field it detects. It
+uses the standard warning response header, see
+this [blog post](https://kubernetes.io/blog/2020/09/03/warnings/) for more
+about API server warnings in general.
```

Comment:
Unknown or duplicate fields are ignored in this level, right?

Reply:
Yes, they are dropped but the warning is added to the response.